sidecar/main.go

```diff
@@ -18,9 +18,10 @@ package main
 
 import (
 	"flag"
-	"time"
+        "time"
```

Comment:
you might have used space instead of tab here, please check

Reply:
yes, I changed to tab now.